logging/bin/getlogs.py

```diff
@@ -15,29 +15,34 @@
 
 def validate_input(dict):
     """Validate the arguments passed by the user to ensure script will function properly"""
-    if args['create-auth']:
+    if dict['create-auth']:
```

Comment:
This change made me think a little bit about some of the var names we are using in this script. Lets rename this list to something more descriptive. (maybe something like `arg_list` or whatever you feel can convey what its being used for)